react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
 
 
     <application
+      android:name=".MainApplication"
```

Comment:
RN android split up the MainActivity slightly: see https://github.com/facebook/react-native/releases/tag/v0.29.0